src/styles/blocks/_catalog.scss

```
@@ -11,9 +11,15 @@
     align-items: center;
     width: 200px;
     height: 300px;
-    border: 1px solid #ccc;
+    border: 1px solid $border-color;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: $transition-duration;
```

```suggestion
    transition: color $transition-duration transform $transition-duration;
```
Sorry didn't notice it during previous review